tasks/README.md

```diff
@@ -6,4 +6,11 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task305_QUAREL_correct_answer_generation` | Given a sentence and a question, write the correct answer based on the sentence. | Answer Generation
+`task306_QUAREL_incorrect_answer_generation` | Given a sentence and a question, write the incorrect correct answer based on the sentence. | Incorrect Answer Generation
+`task307_QUAREL_correct_option_generation` | Given a sentence and a question, choose the correct option based on the sentence. | Answer Generation
+`task308_QUAREL_incorrect_option_generation` | Given a sentence and a question, choose the incorrect option based on the sentence. | Incorrect Answer Generation
```

Comment:
"Choose the option number instead of exact answer". Else it looks same

Reply:
done